### PR TITLE
feat: add an initial testing banner

### PR DESCRIPTION
**Pull request by Shou**
Initial test banner.

### CODE REVIEW

**Comment by craigmulligan** [comment]
Hey @Shou what is this? I'm busy rewriting this site atm so we should probably chat if you adding things?

**Comment by lurch** [comment]
@craig-mulligan I believe this is related to something @alexandrosm requested in flowdock. It's just a standalone file, and is independent of the 'regular' etcher-homepage.

File: banner/index.html [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <meta property="og:title" content="Etcher by Resin.io" />
+    <meta property="og:description" content="Burn images to SD cards and USB drives, safe and easy." />
+    <meta property="og:image" content="../static/images/interface.png" />
+    <link rel="shortcut icon" type="image/png" href="../static/images/etcher.ico"/>
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    <title>Etcher by resin.io</title>
+    <!-- Bootstrap -->
+    <link href="../static/styles/main.css?v=1.5" rel="stylesheet">
+    <!-- fonts -->
+    <script src="https://use.typekit.net/lzw7tre.js"></script>
```
Comment (lurch) [inline banner/index.html]: @Shou could we make this _not_ depend on 3rd-party sites? Or would that break the styling?

File: banner/index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta property="og:url" content="http://www.etcher.io" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Etcher by Resin.io" />
+    <meta property="og:description" content="Burn images to SD cards and USB drives, safe and easy." />
+    <meta property="og:image" content="../static/images/interface.png" />
+    <link rel="shortcut icon" type="image/png" href="../static/images/etcher.ico"/>
```
Comment (lurch) [inline banner/index.html]: Only a minor nit, but I guess we don't really need all these meta-tags, if this page isn't going to be indexed by web-crawlers etc. ?

**Comment by craigmulligan** [comment]
@Shou, I've update the whole site so this PR will be out of date however I've added a page here: https://etcher.io/banner/.

Let me know when you work on this again, because building it locally requires some symlinking while we wait for some dependent PRs to be merged.

**Pull request closed without merging**